actions/actions-plugins/actions-kafka/src/main/java/org/hawkular/alerts/actions/kafka/KafkaPlugin.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.actions.kafka;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.hawkular.alerts.actions.api.ActionMessage;
+import org.hawkular.alerts.actions.api.ActionPluginListener;
+import org.hawkular.alerts.actions.api.ActionPluginSender;
+import org.hawkular.alerts.actions.api.ActionResponseMessage;
+import org.hawkular.alerts.actions.api.Plugin;
+import org.hawkular.alerts.actions.api.Sender;
+import org.hawkular.alerts.api.json.JsonUtil;
+import org.hawkular.alerts.api.model.action.Action;
+import org.hawkular.alerts.log.AlertingLogger;
+import org.hawkular.commons.log.MsgLogging;
+import org.hawkular.commons.properties.HawkularProperties;
+
+import com.bazaarvoice.jolt.Shiftr;
+
+/**
+ * Action Kafka Plugin.
+ *
+ * It processes Actions writing Event/Alerts into an ElasticSearch system.
+ *
+ * It supports optional JOLT Shiftr transformations to process Events/Alerts into custom JSON formats.
+ *
+ * i.e.  {"tenantId":"tenant", "ctime":"timestamp", "dataId":"dataId","context":"context"}
+ *
+ * https://github.com/bazaarvoice/jolt/blob/master/jolt-core/src/main/java/com/bazaarvoice/jolt/Shiftr.java
+ *
+ * @author Jay Shaughnessy
+ * @author Lucas Ponce
+ */
+@Plugin(name = "kafka")
+public class KafkaPlugin implements ActionPluginListener {
+    public static final String PLUGIN_NAME = "kafka";
+
+    /**
+     * "url" property is used to indicate the Kafka server or servers to connect.
```

Comment:
What is this "url" javadoc? I think this might be something that was left over after a cut-n-paste? Probably should be deleted.